health_cards.gemspec

```diff
@@ -5,16 +5,30 @@ require 'rake'
 Gem::Specification.new do |spec|
   spec.name          = 'health_cards'
   spec.version       = HealthCards::VERSION
-  spec.authors       = ['Reece Adamson']
+  spec.authors       = ['Reece Adamson',
+                        'Samuel Sayer',
+                        'Stephen MacVicar',
+                        'Neelima Karipineni',
+                        'Daniel Lee',
+                        "Mick O'Hanlon",
+                        'Priyank Madria',
+                        'Shaumik Ashraf']
   spec.email         = ['radamson@mitre.org']
-  spec.summary       = 'Create Health Cards using FHIR and Verifiable Credentials'
-  spec.description   = 'Create Health Cards using FHIR and Verifiable Credentials'
+  spec.summary       = <<~TEXT
+    Create SMART Health Cards using FHIR and Verifiable Credentials for secure and decentralized#{' '}
```

Comment:
Are interpolated spaces at the end of lines really necessary?

Reply:
This for rubocop, but now rubocop ignores the gem spec so I removed it